### ISSUE TITLE
VZD: Associate non-CR3 records to Locations

### ISSUE THREAD

**Issue by JaceDeloney** [cityofaustin/atd-vz-data#483]
In order to complete #450, we need to associate non-CR3 records to Locations. This requires creating a new association table similar to atd_txdot_crash_locations.

**Comment by sergiogcx** [comment]
@JaceDeloney Regular (actual CR3) records aren't being associated to locations either, should we extend the scope of this epic to also include CR3s?

**Comment by mateoclarke** [comment]
I think we are waiting on the VZ team to give us a new feature layer of locations. Once we have that that feature layer should be pretty stable. We'll need to re-associate crashes (CR3 & non-CR3s) to those locations, and then we should start automating that association for incoming records and edited coordinates on existing records.

**Comment by mateoclarke** [comment]
So I think the answer is yes, but maybe we open a new issue for that:
> automating the location association for incoming records and edited coordinates on existing records.

**Comment by mateoclarke** [comment]
The PR associated to this is not performant enough to run on production. Going to reopen and reproach this problem.

**Comment by mateoclarke** [comment]
Complete in staging. Need to re-run on the production DB now.
